test/functional/columns/vector/vector-similarity.ts

```diff
@@ -0,0 +1,139 @@
+import "reflect-metadata"
+import { expect } from "chai"
+import { DataSource } from "../../../../src/data-source/DataSource"
+import {
+    closeTestingConnections,
+    createTestingConnections,
+    reloadTestingDatabases,
+} from "../../../utils/test-utils"
+import { Post } from "./entity/Post"
+
+describe("columns > vector type > similarity operations", () => {
```

Comment:
I put SAP vector tests under `test/functional/database-schema/vectors/sap` so that they're next to the column types. Could you move it there as well?